person.rb

```diff
@@ -1,14 +1,15 @@
 require_relative 'nameable'
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
 
   def initialize(name: 'Unknown', age: 0, parent_permission: true)
     super()
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
```

Comment:
- [ ] Please fix the following error that occurs whenever the user tries to create a student.

![image](https://github.com/yusufyusie/School-Library/assets/62290545/bb408dd5-b9df-45e8-b86b-b7045420bcc2)

- [ ] A similar error appears when creating a teacher. Kindly fix that as well.

![image](https://github.com/yusufyusie/School-Library/assets/62290545/77ebdbc6-6653-4b25-bd1c-fa112f824e6b)